addons/l10n_bf/data/account_tax_report_data.xml

```diff
@@ -400,7 +400,7 @@
                     <record id="account_tax_report_line_bf_net_vat_to_pay_balance" model="account.report.expression">
                         <field name="label">balance</field>
                         <field name="engine">aggregation</field>
-                        <field name="formula">BF_GROSS_TOTAL.balance - (BF_DEDUCTIBLE.balance + BF_CREDIT_REPORTED.balance - BF_CREDIT_ASKED.balance + BF_CANCELLED.balance + BF_OTHER_DEDUCTION.balance + BF_UNPAID_CREDIT.balance)</field>
+                        <field name="formula">BF_GROSS_TOTAL.balance - (BF_DEDUCTIBLE.balance + BF_CREDIT_REPORTED._applied_carryover_balance - BF_CREDIT_ASKED.balance + BF_CANCELLED.balance + BF_OTHER_DEDUCTION.balance + BF_UNPAID_CREDIT.balance)</field>
```

Comment:
Following my comment above, in those formulas, you should keep calling balance

(multiple occurrences, in other files as well)